Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py

```
@@ -0,0 +1,85 @@
+    res = demisto.executeCommand('WordTokenizerNLP', {'value': text,
+                                                      'hashWordWithSeed': demisto.args().get('hashSeed')})
+    if is_error(res[0]):
+        return_error(res[0]['Contents'])
+    tokenized_text_result = res[0]['Contents']
+    input_text = tokenized_text_result['hashedTokenizedText'] if tokenized_text_result.get('hashedTokenizedText') else \
+        tokenized_text_result['tokenizedText']
+    filtered_text, filtered_text_number_of_words = demisto_ml.filter_model_words(input_text, model)
+    if filtered_text_number_of_words == 0:
+        return_error("The model does not contains any of the input text words")
+
+    explain_result = demisto_ml.explain_model_words(model,
+                                                    input_text,
+                                                    float(demisto.args().get('labelProbabilityThreshold', 0)),
+                                                    float(demisto.args().get('wordThreshold', 0)),
+                                                    int(demisto.args()['topWordsLimit']))
+
+    if tokenized_text_result.get('hashedTokenizedText'):
+        hash_word_to_plain = dict(
+            zip(tokenized_text_result['hashedTokenizedText'].split(" "),
+                tokenized_text_result['tokenizedText'].split(" ")))
+        explain_result['PositiveWords'] = map(lambda x: hash_word_to_plain[x], explain_result['PositiveWords'])
+        explain_result['NegativeWords'] = map(lambda x: hash_word_to_plain[x], explain_result['NegativeWords'])
+    explain_result['OriginalText'] = tokenized_text_result['originalText']
+    explain_result['TextTokensHighlighted'] = tokenized_text_result['tokenizedText']
+
+    res = demisto.executeCommand('HighlightWords', {'text': tokenized_text_result['tokenizedText'],
+                                                    'terms': ",".join(explain_result['PositiveWords'])})
+    res = res[0]
+    if not is_error(res):
```

dont you want to return the error else ?